playlists/excel.yaml

```diff
@@ -9,6 +9,7 @@
   group: Basics
   api_set:
     ExcelApi: 1.1
+  type: public
```

Comment:
It may not be a good idea to get rid of host, you would need to verify if Script Lab has a dependency on this property or you will break compatibility with this repo.